.github/workflows/test.yml

```diff
@@ -44,6 +44,6 @@ jobs:
       - name: Install dependencies
```

Comment:
We will also need to change `short` job as well since we moved to `uvu` and need hacks to support old Node.js versions

Reply:
For example? ESM is not used in the project. Tests are [passing](https://github.com/browserslist/browserslist/runs/4332181761?check_suite_focus=true)

Reply:
Did you changed this line?

https://github.com/browserslist/browserslist/blob/main/.github/workflows/test.yml#L47

Reply:
Sure!

https://github.com/win0err/browserslist/blob/feature/uvu/.github/workflows/test.yml#L47

Reply:
Sorry. I just woke up :)